### ISSUE TITLE
FDBM on GPU

### ISSUE THREAD

**Issue by NikoOinonen** [Probe-Particle/ppafm#35]
Just opening this issue to say that I'm working on making an OpenCL implementation of the full density-based model (FDBM) according to https://doi.org/10.1021/acsnano.8b08209 and https://doi.org/10.1021/acs.jcim.1c01323. The CPU-only version already exists via the scripts `conv_rho.py` and `relaxed_scan_PVE.py`. I will make pull request when it's ready.

blocked by #112

**Comment by ondrejkrejci** [comment]
Formulas for damping: https://manual.q-chem.com/5.2/Ch5.S7.SS2.html if necessary.

**Comment by NikoOinonen** [comment]
So the odd vdW term was for damping. It is not very ideal, so a new issue was created for fixing that at #48 .
